jest.config.js

```diff
@@ -37,7 +37,7 @@ module.exports = {
             branches: 95,
             functions: 98,
             lines: 99,
-            statements: 98
+            statements: 99
```

Comment:
Will you increase coverage with another PR?

Reply:
this patch already increases the coverage 😉